frontend/src/types/signIn.ts

```diff
@@ -1,10 +1,15 @@
+const SIGN_STYLE_PROPS = {
+  focusedColor: '#1D1D1D',
+  basicColor: '#C6C6C9',
+  loginLabel: 'login',
+  passwordLabel: 'password',
+} as const;
```

Comment:
why is this constant not in the constants folder? as far as I understand it is not related to types